code/defines/obj/weapon.dm

```diff
@@ -636,4 +636,4 @@
 	icon = 'icons/obj/stock_parts.dmi'
 	icon_state = "spring"
 	origin_tech = list(TECH_ENGINEERING = 1)
-	matter = list(DEFAULT_WALL_MATERIAL = 40)
\ No newline at end of file
+	matter = list(DEFAULT_WALL_MATERIAL = 40)

```

Comment:
Why was this file touched, exactly?

Reply:
I realized that the fork had my hypospray cane still there, so I removed it. Since a few lines of code were removed, that line got bumped back up to it's original spot.

Reply:
Eh.